fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalAggregationOperator.java

```diff
@@ -153,10 +153,10 @@ public Map<ColumnRefOperator, ScalarOperator> getColumnRefMap() {
     public Map<ColumnRefOperator, ScalarOperator> getLineage(
             ColumnRefFactory refFactory, ExpressionContext expressionContext) {
         Map<ColumnRefOperator, ScalarOperator> columnRefMap = Maps.newHashMap();
-        columnRefMap.putAll(getColumnRefMap());
         if (projection != null) {
             columnRefMap.putAll(projection.getColumnRefMap());
         }
+        columnRefMap.putAll(getColumnRefMap());
```

Comment:
What's the difference

Reply:
rojection.getColumnRefMap() may have the same keys with getColumnRefMap(). I want getColumnRefMap() to replace the same keys of projection.getColumnRefMap()